employee_service/src/main.py

```diff
@@ -72,6 +72,8 @@ def update_data(empid):
         return Response("{'error': 'Erro addding the item'}", mimetype='application/json', status=500)
     return Response(json.dumps(update_emp_res), mimetype='application/json', status=200)
 
+
+
```

Comment:
print statement at line 50 can be ignored
`print("fwqf")`

Even the comments can be ignored